newrelic/hooks/external_botocore.py

```diff
@@ -120,6 +121,9 @@ def create_chat_completion_message_event(
             "ingest_source": "Python",
         }
 
+        if settings.ai_monitoring.record_content.enabled:
+            chat_completion_message_dict.update({"content": message.get("content", "")})
```

Comment:
I just noticed this is using update everywhere instead of assignment. Let's use assignment. Update is much slower performance wise and since we are only adding one new key value pair it doesn't make sense to use update here.